charts/xinference/templates/deployment-supervisor.yaml

```diff
@@ -86,8 +86,8 @@ spec:
         - name: {{ $key }}
           value: {{ $value }}
 {{- end }}
-{{- if .Values.config.xinference_image }}
-        image: {{ .Values.config.xinference_image | quote }}
+{{- if .Values.xinferenceSupervisor.supervisor.image }}
```

Comment:
Can we support them both? when setting a single image, unified for supervisor and worker, if not, users can specify them separately.

Reply:
Good point, PR updated